public/app/plugins/datasource/prometheus/querybuilder/shared/parsingUtils.ts

```diff
@@ -115,8 +115,8 @@ export function makeBinOp(
  * @param cur
  * @param type
  */
-export function getAllByType(expr: string, cur: SyntaxNode, type: string): string[] {
-  if (cur.name === type) {
+export function getAllByType(expr: string, cur: SyntaxNode, type: number | string): string[] {
```

Comment:
Nit pick suggestion: I would add a comment that type can be string or number because Loki currently uses hard coded names instead of `lezer-logql` imported types. And that we should remove `string` type for `type` when Loki removes hardcoded string in favour of ids.

Reply:
@ivanahuckova Added!